src/NzbDrone.Core/Datastore/Migration/163_download_client_per_indexer.cs

```diff
@@ -0,0 +1,14 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(163)]
```

Comment:
This is already used in another PR:

https://github.com/Sonarr/Sonarr/pull/4815